tests/cephadm/test_prepare_host.py

```diff
@@ -0,0 +1,27 @@
+from cli.cephadm.cephadm import CephAdm
+
+
+class PrepareHostError(Exception):
+    pass
+
+
+def run(ceph_cluster, **kw):
+    """Verify prepare host command
+    Args:
+        kw: test data
+            e.g:
+            - test:
+                name: test_prepare-host_command
+                desc: Check-host/prepare-host for cephadm use/host configuration
+                polarion-id: CEPH-83573789
+                module: test_prepare_host.py
+                config:
+                    result: Host looks OK
+    """
+    config = kw.get("config")
+    node = ceph_cluster.get_nodes(role="mon")[0]
+    result = CephAdm(node).prepare_host()
```

Comment:
prepare_host [1] function require parameters, missing to pass ?
[1] https://github.com/red-hat-storage/cephci/pull/2167/files#diff-b8cd2f30f4dd9fabfedeb80881699702a81220bb949f17a3f1105fddb49975a0R45

Reply:
@vamahaja 

As per the test case we need to run command without any parameter.

Reply:
ok